docs/variants/msi_z690/recovery.md

```diff
@@ -44,3 +44,66 @@ $ flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.13
 ```
 
 * First boot after the recovery process is significantly longer
+
+### SMBIOS unique data recovery
+
+#### Serial number
+
+SMBIOS specification defines two spaces for serial number: the system serial
+number and baseboard serial number. The original MSI PRO Z690-A firmware
+provides only baseboard serial number.
+
+In case you have lost your serial number in the process of flashing Dasharo or
+newer MSI firmware, there is a way to retrieve it. The board has a QR code
+imprinted between the chipset heatsink and dPGU PCIe slot:
+
+![](/images/msi_sn_qr.jpg)
+
+If you read the QR code with your smartphone you will get the full serial
+number. The serial number has the format `07D25xx_LyzEaaaaaa` where:
+
+* `07D25` - is the board model, i.e. MS-7D25 for this particular board
+* `xx` is the mainboard revision which should match the revision imprinted
+  between the M2_1 slot and dGPU slot. E.g. `xx=11` means VER:1.1
+* `yz` is the manufacturing date in hex, i.e. `y` is the month, `z` is the
+  year, for example `A1` means **October** 202**1**, `12` means **January**
+  202**2**
+* `aaaaaa` is the unique 6-digit number which is imprinted under the serial
+  number QR code
+
+#### System UUID
+
+SMBIOS specification defines a field for unique system identification with a
+special number called UUID (Universally Unique IDentifier). UUID is specified
+by [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122). MSI firmware
+provides the system UUID in the SMBIOS system information structure.
+
+The problem with UUID is that it cannot be recovered if the backup binary or
+SMBIOS logs are lost. You can backup the SMBIOS information with our
+[fwdump-docker](https://github.com/3mdeb/fwdump-docker). The `dmidecode.log`
+will have all the necessary information. Things we know about MSI system UUID:
+
+- UUID format is as follows: `33221100-5544-7766-8899-AABBCCDDEEFF`, the hex
+  numbers represent the order of bytes in memory for the little-endian format
+  as required by SMBIOS
+- MSI UUIDs do not conform to any of the RFC 4122 UUID variants/versions (the
+  bits responsible for UUID version and variant identification are not constant
+  across multiple boards)
+- the last octet group `AABBCCDDEEFF` is equal to the MAC address of the
+  on-board Intel i225 Ethernet, so **be sure to NOT share the UUID** with
```

Comment:
is MAC really **that** sensitive?

Reply:
It can be some privacy/info leak, please note Qubes OS and other distributions provide MAC addres randomization https://github.com/Qubes-Community/Contents/blob/master/docs/privacy/anonymizing-your-mac-address.md

I believe this is correct point and DTS should consider privacy respecting HCL which drops that part from UUID. Even this discussion and section can be linked in commits for future reference.

Reply:
It depends on what HCl is used for. If HCL is used to provide the machine owner a backup of info, then it shouldn't drop that information. The info should be stripped only when sending to our mailboxes.

Reply:
https://github.com/Dasharo/dasharo-issues/issues/92